loadrunner_professional/assets/logs/loadrunner_professional.yaml

```diff
@@ -0,0 +1,71 @@
+id: loadrunner-professional
```

Comment:
The id needs to match the name of the file. In this case the file should be called `loadrunner-professional` (or the other way around, the id of the integration should be `loadrunner_professional`).